queries.sql

```diff
@@ -1,3 +1,63 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+SELECT *
+FROM animals
+WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+SELECT *
```

Comment:
- [ ] Kindly make sure that you're following the project requirements which dictate that the species should be set to 'unspecified' and then rollbacked. Kindly revisit the requirements and implement the changes. See the example down below

```sql
UPDATE animals
   SET species = 'unspecified';

   SELECT species from animals;
   ROLLBACK;

   SELECT species from animals;
```